dbLayer/src/test/java/com/plhal/ares/dblayer/EmbeddedPostgresTest.java

```diff
@@ -0,0 +1,51 @@
+package com.plhal.ares.dblayer;
+
+import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+/**
+ * Class for application testing.
+ */
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = App.class)
```

Comment:
Tady se mi nelibi, ze kvuli autokonfiguraci databaze a tomuto testu se nahazuje "spring boot applikace", kterou nikdo nepotrebuje a nepouzije. Nicmene pro potrebu teto zkusebni aplikace je to akceptovatelne.